aspnetcore/security/authentication/identity.md

```diff
@@ -27,11 +27,20 @@ In this topic, you'll learn how to use ASP.NET Core Identity to add functionalit
 1.  Create an ASP.NET Core Web Application project with Individual User Accounts.
 
     # [Visual Studio](#tab/visual-studio)
-    In Visual Studio, select **File** -> **New** -> **Project**. Select the **ASP.NET Web Application** from the **New Project** dialog box. Selecting an ASP.NET Core **Web Application(Model-View-Controller)** for ASP.NET Core 2.x with **Individual User Accounts** as the authentication method.
+    In Visual Studio, select **File** -> **New** -> **Project**. Select **ASP.NET Core Web Application** and click **OK**. 
```

Comment:
`**File** -> **New** -> **Project**`

Are we using `->` for UI? I thought `>` bewteen. I don't care what it is, as long as we're consistent.

cc @rpetrusha @mairaw  @scottaddie

Reply:
We should be using >. Here's what the style guide says...

> Use a greater-than symbol (>) to separate sequential steps.
Only use this approach when there’s a clear and obvious path through the UI and the selection method is the same for each step. For example, don’t mix things that require opening, selecting, and choosing.
Don’t bold the greater-than symbol. Include a space before and after the symbol.

Reply:
What @scottaddie said 😄